vm/src/exceptions.rs

```diff
@@ -138,6 +140,10 @@ impl ExceptionZoo {
         let import_error = create_type("ImportError", &type_type, &exception_type, &dict_type);
         let module_not_found_error =
             create_type("ModuleNotFoundError", &type_type, &import_error, &dict_type);
+        let file_not_found_error =
+            create_type("FileNotFoundError", &type_type, &import_error, &dict_type);
```

Comment:
The base type should probably be different than `import_error`?

Reply:
Follow up fix in #309